src/test/java/Homework23

```diff
@@ -0,0 +1,18 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+
+    public class HomeWork23 extends BaseTest {
+        @Test
+        public void renamePlaylist() {
+            LoginPage loginPage = new LoginPage(driver);
+            HomePage homePage = new HomePage(driver);
+            loginPage.logIn();
+            homePage.choosePlaylist();
+            homePage.conTextClick();
+            homePage.newName();
+            Assert.assertTrue(homePage.checkPlayList());
+        }
+    }
+
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

`class HomeWork23` is  not java -class

![image](https://user-images.githubusercontent.com/104761482/216807705-4bc2c03c-4ad4-4d77-9a44-a950bad40565.png)

![image](https://user-images.githubusercontent.com/104761482/216807719-cf9b883d-62fa-4312-ad97-83eaacf5426a.png)